src/Npgsql/PostgresMinimalDatabaseInfo.cs

```diff
@@ -25,7 +25,7 @@ class PostgresMinimalDatabaseInfo : PostgresDatabaseInfo
 
         static readonly PostgresBaseType[] Types = typeof(NpgsqlDbType).GetFields()
             .Select(f => f.GetCustomAttribute<BuiltInPostgresType>())
-            .Where(a => a != null)
+            .OfType<BuiltInPostgresType>()
```

Comment:
Ah, this is recognized for nullability? Interesting...

Reply:
But it's slower because `OfType<T>` uses the `is` operator inside.

Reply:
OK, good to know.